src/Microsoft.OData.Core/Json/BufferingJsonReader.cs

```diff
@@ -127,24 +117,21 @@ public JsonNodeType NodeType
         /// Depending on whether buffering is on or off this will return the node type of the last
         /// buffered read or the node type of the last unbuffered read.
         /// </remarks>
-        public object Value
+        public object GetValue()

```

Comment:
Is this a necessary change? Are there any benefits of changing thit to a method?

Reply:
The `Value` property is not a simple a property, it actually performs an I/O operation to read a value from the stream. The change from `Value` property to `GetValue()` method serves to better reflect that this method does some heavy lifting. It also serves to create better parity with the async version, which is `GetValueAsync()`. This change was proposed by @gathogojr